azure-pipelines.yml

```diff
@@ -50,6 +50,21 @@ jobs:
         tasks: 'build'
         options: '-PskipAthena'
 
+ - job: Styleguide
+   pool:
+     vmImage: 'Ubuntu 16.04'
+
+   steps:
+      - script: |
+        sudo apt-get update -y
+        sudo apt-get install clang-format-5.0 -y
+        sudo python3 -m pip install --user wpiformat
```

Comment:
If you're running `pip install` with `--user`, the package is installed in the user's /home directory, so `sudo` shouldn't be needed.